gdk4-wayland/Cargo.toml

```diff
@@ -24,6 +25,8 @@ gio = {git = "https://github.com/gtk-rs/gtk-rs-core", features = ["v2_66"]}
 glib = {git = "https://github.com/gtk-rs/gtk-rs-core", features = ["v2_66"]}
 libc = "0.2"
 wayland-client = {version = "0.28", features = ["use_system_lib"]}
+khronos-egl = "4.1.0"
+xkb = "0.2.1"
```

Comment:
Those should be enabled only if v4_4 is selected?

Reply:
ack